test/google_scraping_web/api/v1/auth_controller_test.exs

```diff
@@ -0,0 +1,49 @@
+defmodule GoogleScrapingWeb.Api.V1.AuthControllerTest do
+  use GoogleScrapingWeb.ConnCase, async: true
+
+  describe "POST create/2" do
+    test "given valid user credentials, returns access token", %{conn: conn} do
+      email = "test@test.com"
+      insert(:user, email: email)
+
+      conn =
+        post(conn, Routes.api_auth_path(conn, :create), %{
+          email: email,
+          password: valid_user_password()
+        })
+
+      assert %{
+               "data" => %{
+                 "attributes" => %{
+                   "email" => ^email,
+                   "token" => _
+                 },
+                 "id" => _,
+                 "relationships" => %{},
+                 "type" => "auth"
+               },
+               "included" => []
+             } = json_response(conn, 200)
```

Comment:
Minor comment, but we can add the asserting value to the right

```elixir
assert json_response(conn, 200) == %{ ... }
```

Reply:
I don't think we can adjust it like this because we are not doing equality assertion here. We are using pattern matching with skipping a few fields. What do you think?